pydatalab/pydatalab/config.py

```diff
@@ -60,12 +60,17 @@ class RemoteFilesystem(BaseModel):
 class ServerConfig(BaseSettings):
     """A model that provides settings for deploying the API."""
 
-    SECRET_KEY: str = Field(os.urandom(12).hex(), description="The secret key to use for Flask.")
+    SECRET_KEY: str = Field(description="The secret key to use for Flask.")
```

Comment:
Is this necessary? This breaks the CI since secret key isn't fixed, and also means local installs get a little more complicated because it will fail if you haven't set the secret key.

Reply:
Alternatively, do we have a "template" config file or .env file? Sorry if I've missed it.

Reply:
> Is this necessary? This breaks the CI since secret key isn't fixed, and also means local installs get a little more complicated because it will fail if you haven't set the secret key.

It kinda is, yeah. Quick reasons: setting a default random value here means that different workers get different keys in deployment, which breaks JWT. Setting a value in the config.json means it should be under version control = bad, and as of now, omitting it from config.json and adding it to .env leads to the default random config taking precedence.

For your local stuff just launch the server with `PYDATALAB_SECRET_KEY="blah"` until we can work out the best way of doing it (IMO just removing it entirely from our Python code would be preferable).

Reply:
Made the default a system-specific hash of platform info. Not exactly secure but probably pretty good (and better than an openly-available default). Could probably find hard to obtain metadata to add that is shared by all potential flask workers, e.g., server uptime, disk usage by a specific temporary file...